### PR TITLE
chore: eslint rule update and readme addition

### PR DESCRIPTION
**Pull request by tarnas14**
Description
---

- add curly braces eslint rule for consistency
- add developer rules to readme.md 

Motivation and Context
---

this is for our development to be more streamlined and consistent, so that the code looks similar across the codebase

How Has This Been Tested?
---

n/a

### CODE REVIEW

**Review by tomaszantas — DISMISSED** [review]
LGTM!

**Comment by tarnas14** [comment]
@corquaid this is a team-level change/agreement, could you also skim through this PR and add your approval if you find it worthy? :D

**Review by corquaid — DISMISSED** [review]
LGTM!

File: applications/launchpad_v2/README.md [hunk 1]
```diff
@@ -57,6 +57,39 @@ $ npm run bundle
...
+
+component JSDoc examples:
+```js
+/**
+ * renders tari button
+ *
+ * @prop {() => void} onClick - event handler for click event
+ * @prop {string} color - text color of the button text
+ */
+const TariButton = ({ onClick }: TariButtonProps) => { ... }
```
Comment (tomaszantas) [inline applications/launchpad_v2/README.md]:
```suggestion
const TariButton = ({ onClick, color }: TariButtonProps) => { ... }
```

Comment (tomaszantas) [reply applications/launchpad_v2/README.md]: Since the `color` is in the JSDoc, then it's good to have it also in the props

**Review by tomaszantas — CHANGES_REQUESTED** [review]
One comment

**Review by tomaszantas — APPROVED** [review]

**Pull request merged**